src/opt.rs

```diff
@@ -106,6 +113,42 @@ pub(crate) enum Target {
     Bench(String),
 }
 
+/// Represents the target of package for example, bin or bench
+#[derive(Debug, PartialEq)]
+pub(crate) enum Package {
+    Default,
+    Package(String),
+}
+
+impl From<Package> for Packages {
+    fn from(p: Package) -> Self {
+        match p {
+            Package::Default => Packages::Default,
+            Package::Package(s) => Packages::Packages(vec![s]),
+        }
+    }
+}
+
+impl Clone for Package {
```

Comment:
Isn't this derivable?

Reply:
I've simplified this in the latest commit.